### DISCUSSION TITLE
[Copy annotation] sometimes it is difficult to annotate same objects on different images, it will be really helpful if we can get a feature that can copy annotations from one image to another, if already exists, please explain.

### DISCUSSION THREAD

**Discussion by vishalmandley** [wkentaro/labelme#1106] [Q&A / Help] [answered]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by wkentaro** [comment, marked as answer]
If you use `keep_prev: true/false` option in `~/.labelmerc` or `labelme --keep-prev` or the menu bar, it will keep the polygons in the previous frame.
